### PR TITLE
One monolithic factory-entries data product

### PR DESCRIPTION
**Pull request by knoepfel**
Instead of one data product for each factory entry type, this PR merges all of those products into a monolithic data product called `"Factory_Entries"`.

### CODE REVIEW

**Comment by StevenCTimm** [comment]
Should also check glideinwms/glide_frontend_element.py to make sure there's nothing in there.  Otherwise this looks good.

File: src/decisionengine_modules/glideinwms/publishers/fe_group_classads.py [hunk 1]
```diff
@@ -50,13 +49,19 @@ def create_invalidate_constraint(self, requests_df):
...
 
     def dataframe_for_entrytype(self, allow_type, datablock):
-        requests_df = datablock.get("glideclient_manifests")
         facts_df = datablock.get("de_logicengine_facts")
 
-        data_product_name = f"Factory_Entries_{allow_type}"
+        requests_df = datablock.get("glideclient_manifests")
+        if requests_df.empty:
+            return pandas.DataFrame()
+
```
Comment (shreyb) [inline src/decisionengine_modules/glideinwms/publishers/fe_group_classads.py:57]: Not a blocker, but is it fairly trivial to add a test to make sure that if the `glideclient_manifests` data block is empty, we get an empty `pandas.DataFrame()`?

**Review by shreyb — APPROVED** [review]
Approved, but I had one comment to see if it's possible to increase coverage for the lines added in fe_group_classads.py.

File: src/decisionengine_modules/glideinwms/publishers/fe_group_classads.py (hunk 1 above)
Comment (knoepfel) [reply src/decisionengine_modules/glideinwms/publishers/fe_group_classads.py:57]: Yes, I'll find a way to cover this.
